recipes/lzfse/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "lzfse" %}
+{% set version = "2017.03.08" %}
+{% set sha256hash = "68193df40e8bc8eb87ee7ec47f2da4be64d1c172d1ce558d38aefd0bdcc0ed51" %}
+{% set ghcommit = "88e2d2788b4021d0b2eb9fe2d97352ae9190f128" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ ghcommit }}.zip
+  sha256: {{ sha256hash }}
+
+build:
+  number: 0
+  # We can't compile c99 lzfse source in VS2008.
+  skip: True  # [win and py27]
+  features:
+    - vc14  # [win and py3k]
```

Comment:
Would recommend not using `py3k` and using `(py35 or py36)` as this will probably need to be changed for Python 3.7 anyways.